ui/app/templates/vault/cluster/secrets/backend/list.hbs

```diff
@@ -39,21 +39,46 @@
         {{/if}}
       </ToolbarFilters>
 
-      <ToolbarActions>
-        <ToolbarSecretLink
-          @secret=""
-          @mode="create"
-          @backend={{this.backendModel.id}}
-          @type="add"
-          @queryParams={{hash initialKey=(or this.filter this.baseKey.id) itemType=this.tab}}
-          data-test-create-secret-link
-        >
-          {{options.create}}
-        </ToolbarSecretLink>
-      </ToolbarActions>
+      {{! Old Engines will keep the existing toolbar actions }}
+      {{#if (get (engines-display-data this.backendType) "isOldEngine")}}
```

Comment:
[nit] it might be nice to have a computed property for this like:
```javascript
get isOldEngine() {
  const engineData = engineDisplayData(this.backendType);
  return engineData?.isOldEngine ?? false;
}
```

this property could be used on lines `43` & `61` like: 
```hbs
{{#if this.isOldEngine}}
```
and 
```hbs
{{#if (not this.isOldEngine)}}
```